src/components/Landing/EventCard/EventCardModal/EventCardModal.scss

```diff
@@ -0,0 +1,94 @@
+@import "../../../../styles/variables.scss";
+@import "../../../../styles/fonts.scss";
+@import "../../../../styles/colors.scss";
+@import "../../../../styles/mixins.scss";
+@import "../../../../styles/animations.scss";
+
+.wrapper {
+    @include flexContainer(center, center);
+    width: $fullWidth;
+    height: $fullHeight;
+    background-color: $backGroundModal;
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 1;
+
+    .container {
+        width: 384px;
+        height: auto;
+        box-shadow: $boxshadow;
+        border-radius: $r16;
+        padding: 20px;
+        margin: 10px;
+        background-color: $white;
+    }
+}
+
+
+.titleText {
+    font-size: $fontsize-24;
+    text-align: center;
+}
+
+
+.inputBlock {
+    display: flex;
+    justify-content: center;
```

Comment:
use mixin

Reply:
We don't have mixin just for display and j-c.